SimpleWAWS/Code/CsmExtensions/CsmSiteExtensions.cs

```diff
@@ -90,7 +105,21 @@ public static partial class CsmManager
             return site;
         }
 
-        public static async Task<Site> LoadPublishingCredentials(this Site site)
+        public static async Task UpdateScmConfig(this Site site)
+        {
+            if (site.Kind=="functionapp")
```

Comment:
also let's add a property like `isFunctionApp` or use an Enum to represent the kind of the app. This way we won't have to do string matching everywhere we want to do something special